### ISSUE TITLE
any of the plyranges "good-first-issue" issues

### ISSUE THREAD

**Issue by mikelove** [tidybiology/tidy-ranges-tutorial#6]

**Comment by KittJonathan** [comment]
Hi ! I could start by working on the plyranges vignette figures

**Comment by nahid18** [comment]
Can anyone provide a PDF version of this: [common-tasks.Rmd](https://github.com/sa-lee/plyrangesWorkshops/blob/master/vignettes/common-tasks.Rmd)? Hopefully I can work on the [plyranges cheatsheet](https://github.com/sa-lee/plyranges/issues/26).

**Comment by mikelove** [comment]
> having trouble installing plyranges

Want to make a support.bioconductor.org post and I'll help out? Post your installation error etc.

**Comment by mikelove** [comment]
(tip: you can copy paste code and enclose it with triple backticks on github, it will render like the following chunk)

```
this is code
```

Looks like `common-tasks.Rmd` was last updated 4 years ago. You can take a look at those lines and see if the functions need updating by cross-referencing the current man pages for plyranges.

**Comment by nahid18** [comment]
@mikelove Found a cheatsheet template from here [CONTRIBUTING.md](https://github.com/rstudio/cheatsheets/blob/main/.github/CONTRIBUTING.md) to work on the `plyranges` cheatsheet task

**Comment by mikelove** [comment]
@KittJonathan yes i think so! you could compute them within an Rmd using plyranges and make basic plots. I can give more pointers if you wanna start a repo.
